backend/src/main/java/edu/netcracker/backend/security/JwtAuthFilter.java

```diff
@@ -77,41 +57,50 @@ protected void doFilterInternal(HttpServletRequest httpServletRequest,
     private void handleExpiredToken(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse) {
         String newAccessToken = createNewAccessTokenIfRefreshTokenIsValid(httpServletRequest);
 
-        if (newAccessToken == null) return;
+        if (newAccessToken == null) {
+            return;
+        }
 
         httpServletResponse.setHeader("New-Access-Token", newAccessToken);
         handleToken(newAccessToken, httpServletRequest);
     }
 
     private String createNewAccessTokenIfRefreshTokenIsValid(HttpServletRequest request) {
         String refreshToken = JwtUtils.getRefreshToken(request);
-        if (!jwtProvider.validateToken(refreshToken) || !jwtProvider.isRefreshToken(refreshToken)) return null;
+        if (!jwtProvider.validateToken(refreshToken) || !jwtProvider.isRefreshToken(refreshToken)) {
+            return null;
+        }
 
         User user = userService.findByUsername(jwtProvider.retrieveSubject(refreshToken));
-        if (isNotMatchedWithUsersRefreshToken(refreshToken, user)) return null;
+        if (isNotMatchedWithUsersRefreshToken(refreshToken, user)) {
+            return null;
+        }
 
         return jwtProvider.generateAccessToken(user);
     }
 
     private void handleToken(String accessToken, HttpServletRequest request) {
```

Comment:
In this method you should manually delete redundant lines

Reply:
Deleted to match code style.